src/scripts/main.js

```diff
@@ -5,4 +5,22 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach(person => {
+  const tableRow = document.createElement('tr');
+
+  tableRow.insertAdjacentHTML(
+    'beforeend',
+    `
+      <td>${person.name}</td>
+      <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
+      <td>${person.born}</td>
+      <td>${person.died}</td>
+      <td>${person.died - person.born}</td>
+      <td>${Math.ceil(person.died / 100)}</td>
+    `
```

Comment:
```suggestion
    `
      <tr>
        <td>${person.name}</td>
        <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
        <td>${person.born}</td>
        <td>${person.died}</td>
        <td>${person.died - person.born}</td>
        <td>${Math.ceil(person.died / 100)}</td>
      </tr>
    `
```
You could just add <tr> tag to your `insertAdjacentHTML ` and there would be no need in 'tableRow' const